### PR TITLE
refactor!: updated secret path to secretName

### PR DESCRIPTION
**Pull request by vli11**
BREAKING CHANGE: updated secret path to secretName

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-mqtt-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

## Testing Instructions
make test
make docker

### CODE REVIEW

**Review by jim-wang-intel — APPROVED** [review]
LGTM

File: cmd/res/configuration.toml [hunk 1]
```diff
@@ -5,8 +5,8 @@ LogLevel = "INFO"
   # InsecureSecrets are required for when Redis is used for message bus
   [Writable.InsecureSecrets]
     [Writable.InsecureSecrets.MQTT]
-    path = "credentials"
-      [Writable.InsecureSecrets.MQTT.Secrets]
+    secretName = "credentials"
```
Comment (lenny-intel) [inline cmd/res/configuration.toml:8]:
```suggestion
    SecretName = "credentials"
```

**Review by lenny-intel — CHANGES_REQUESTED** [review]

**Review by lenny-intel — COMMENTED** [review]
LGTM  once tested

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**